database/src/main/dataset_schema/heads.ddl

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+-- DROP TABLE dataset_schema.heads;
```

Comment:
```suggestion
DROP TABLE IF EXISTS dataset_schema.heads;
```
And the same elsewhere, perhaps?

Reply:
`IF EXISTS` sure.
But it's commented out on purpose, so an accidental run won't result in data loss, just an error of trying to create an already existing table.